ReactSkia/components/RSkComponentActivityIndicator.cpp

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (C) 1994-2022 OpenTV, Inc. and Nagravision S.A.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+#include "include/core/SkPaint.h"
+#include "include/core/SkPath.h"
+
+#include <ReactSkia/activityindicator/react/renderer/components/activityindicator/ActivityIndicatorProps.h>
+#include "ReactSkia/components/RSkComponentActivityIndicator.h"
+#include "ReactSkia/views/common/RSkConversion.h"
+
+#include "ReactSkia/utils/RnsLog.h"
+
+#define ACTIVITY_INDICATOR_STROKE_WIDTH_RATIO             12.5
```

Comment:
JFMI: 12.5 % is decided w.r.t iOS ?

Reply:
Taken WEB as reference and manually calculated the stroke width.